src/vs/platform/backup/electron-main/backupMainService.ts

```diff
@@ -51,11 +51,6 @@ export class BackupMainService implements IBackupMainService {
 	}
 
 	public registerWindowForBackups(windowId: number, isEmptyWorkspace: boolean, backupFolder?: string, workspacePath?: string): void {
```

Comment:
That was my thinking, didn't we want to keep that check outside though due to the config in `IEnvironmentService` potentially being stale or something?